e2e-tests/native/wdio-browserstack.conf.ts

```diff
@@ -0,0 +1,74 @@
+import {execSync} from 'child_process'
```

Comment:
Is this copied from web? Looks familiar

Reply:
yep, it is

Reply:
Do we need both or could we use just one for both projects?

Reply:
I'm fine with both, copying or keeping it separate. There will be only 2 configs and 2 copies are still fine imo.

Reply:
For now I will leave it where it is. Due to @maxammann argumentation